src/server/api/controllers/admin/EventsController.ts

```diff
@@ -75,6 +75,17 @@ export class EventsController {
     return newEvent;
   }
 
+  @Post('/edit/:eventId')
+  @UseBefore(RoleAuth(Role.ROLE_ADMIN))
+  async editExistingEvent(
+    @Param('eventId') id: string,
+    @BodyParam('event') event: RegisterEventRequest,
+    @UploadedFile('logo', { options: Uploads, required: false }) logo?: Express.Multer.File,
+  ) {
+    await this.EventService.editExistingEvent(id, event, logo ? logo.path : undefined);
+    return SuccessResponse.Positive;
```

Comment:
This does not make sense - your frontend code seems to rely on this API responding with an event

Reply:
Nice catch! Guess I missed it since getting the event back isn't mission-critical (mostly used for logging). I'll add that now.

Reply:
See: 87d4e2b. Ended up scrapping any front-end code that required a response instead. A `200` response now indicates that the update was successful and a subsequent `GET` call will return the new resource.